MarsCompetition/MarsCompetition/Test/CertificationTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using AventStack.ExtentReports;
+using MarsCompetition.Pages;
+using MarsCompetition.TestModel;
+using MarsCompetition.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsCompetition.Test
+{
+    public class CertificationTest : CommonDriver
+    {
+#pragma warning disable CS8618
+
+        private CertificationPage certificationPageObj = new CertificationPage();
+        private ExtentReports extent;
+        [Test, Order(1)]
+        public void AddCertification_Test()
+        {
+            CommonDriver driver = new CommonDriver();
+            driver.InitializeExtentReports();
+            string testName = "AddCertification";
+            driver.SetupTest(testName);
+            string sFile = "AddCerPosdata.json";
```

Comment:
you have not covered all the tests that have be suggested to do in the task 

you are missing some tests please look at the task spec!